webapp/src/share.tsx

```diff
@@ -172,20 +173,22 @@ pxt extract ${url}`;
                         : undefined }
                     { url && ready ? <div>
                         <p>{lf("Your project is ready! Use the address below to share your projects.") }</p>
-                        <sui.Input class="mini" readOnly={true} lines={1} value={url} copy={true} selectOnClick={true}/>
+                        <sui.Input id="projectUri" class="focused mini" readOnly={true} lines={1} value={url} copy={true} selectOnClick={true} aria-describedby="projectUriLabel"/>
+                        <label htmlFor="projectUri" id="projectUriLabel" className="accessible-hidden">{lf("This is the read-only internet address of your project. You can copy it by pressing Control C.")}</label>
```

Comment:
Nope. It's Cmd C on Mac.. Can we assume an accessible user that is using a screenreader knows how to Copy something with their native app.